fastly/data_source_fastly_tls_certificate_ids_test.go

```diff
@@ -19,17 +19,14 @@ func TestAccFastlyDataSourceTlSCertificateIDs(t *testing.T) {
 	cert = strings.ReplaceAll(cert, "\n", `\n`)
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:  func() { testAccPreCheck(t) },
-		Providers: testAccProviders,
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviders,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccFastlyDataSourceTLSCertificateIDSConfig_resources(name, key, cert),
-			},
-			{
-				Config: testAccFastlyDataSourceTLSCertificateIDSConfig_resourcesAndData(name, key, cert),
```

Comment:
Can you clarify why this test check is being removed?

Reply:
The configuration has been combined into the first step and the use of depends_on allows the data source to be created after the resources were defined.  So this one is just removing the "result" output.